clippy_lints/src/loops/never_loop.rs

```diff
@@ -228,6 +228,7 @@ fn never_loop_expr(expr: &Expr<'_>, ignore_ids: &mut Vec<HirId>, main_loop_id: H
     }
 }
 
+#[clippy::dump]
```

Comment:
Leftover?

Reply:
Yes, it is a leftover, thanks for pointing it out.